test/resources/orderCreation.ts

```diff
@@ -0,0 +1,28 @@
+import { Wallet, Contract, BigNumber } from "ethers";
+
+import { Order } from "../../src/js/orders.spec";
+
+export const orderAlwaysValid = function (
```

Comment:
nit: This should just be a function and not a closure. There is no need to capture the `this` context here:
```suggestion
export function orderAlwaysValid(
```